src/Messages/Interactions/TSOutgoingMessage.h

```diff
@@ -12,9 +12,17 @@ NS_ASSUME_NONNULL_BEGIN
 @interface TSOutgoingMessage : TSMessage
 
 typedef NS_ENUM(NSInteger, TSOutgoingMessageState) {
+    // The message is either:
+    // a) Enqueued for sending.
+    // b) Waiting on attachment upload(s).
+    // c) Being sent to the service.
     TSOutgoingMessageStateAttemptingOut,
+    // The failure state.
     TSOutgoingMessageStateUnsent,
+    // The message has been sent to the service, but not received by any recipient client.
     TSOutgoingMessageStateSent,
+    // The message has been sent to the service, but not received by at least one recipient client.
```

Comment:
I think you mean:

> but not received by at least one recipient client.
~~but not~~ *and* received by at least one recipient client.